### PR TITLE
Update go2rtc version to 1.8.4

### PR DESCRIPTION
**Pull request by skrashevich**
This commit updates the go2rtc package version from 1.8.2 to 1.8.4. The changes include updates to the Makefile and digests files. The changelog has also been updated to reflect the new features and bug fixes introduced in this version. These include improved support for incoming MJPEG, fixes for https source in some cases, and the addition of new GoPro source for streaming from GoPro cameras among others.

## Description

Fixes # 

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by skrashevich** [comment]
up

File: spk/go2rtc/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SPK_NAME = go2rtc
-SPK_VERS = 1.8.2
+SPK_VERS = 1.8.4
 SPK_REV = 2
```
Comment (mreid-tt) [inline spk/go2rtc/Makefile:3]: The `SPK_REV` should increment if you are publishing a new version

File: spk/go2rtc/Makefile [hunk 2]
```diff
@@ -10,7 +10,7 @@ UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 MAINTAINER = skrashevich
 DESCRIPTION = Ultimate camera streaming application with support RTSP, RTMP, HTTP-FLV, WebRTC, MSE, HLS, MP4, MJPEG, HomeKit, FFmpeg, etc
 DISPLAY_NAME = go2rtc
-CHANGELOG = "Add expr source<br />Add support multiline exec source<br />Add IPv6 support to WebRTC <br />Add more ffmpeg transcoding presets<br />Add support host candidate config<br />Increase buffer for RTP producers<br />Update WebRTC candidates logic <br />Fix pixel format for h264 vaapi hardware transcoding<br />Fix ngrok support<br />Fix wrong SDP for MERCURY camera <br />Fix panic for HomeKit source without SRTP module <br />Fix raw conn for https client (wss transport for rtsp source)<br />Fix memory overflow on bad RTSP sources "
+CHANGELOG = "Add support ADTS to magic producer. Improve support incoming MJPEG in some cases. Fix https source in some cases. Fix api port for homekit module. Add new GoPro source for streaming from GoPro cameras  Add support unix socket for api module  Add support incoming H264 bitstream Fix https source   Fix support Tapo C210 firmware v1.3.9  Fix homekit source panic on stop producer Fix panic for broken RTP with AAC  Fix digest auth in some cases"
```
Comment (mreid-tt) [inline spk/go2rtc/Makefile:13]: For your changelog it is recommended that you separate discrete lines with numbers and `<br/>` line breaks as in the example shown in [Makefile variables](https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables#available-variables-in-spk-makefiles).

**Comment by mreid-tt** [comment]
> up

Looking forward to your additional commits to address the points raised.

**Review by mreid-tt — APPROVED** [review]

**Pull request merged**